src/content/docs/network-performance-monitoring/get-started/network-monitoring-guidelines.mdx

```diff
@@ -0,0 +1,330 @@
+---
+title: Network monitoring guidelines for KTranslate containers
```

Comment:
@thezackm What do you think about this title?

Reply:
let's use something more like `Architectural recommendations for network monitoring containers`

I hesitate to use the `KTranslate` name too much since we don't really market that specifically. Also; the intention of this doc is to give a best-practices recc for deploying our network monitoring solution.